Modix/Modix.csproj

```diff
@@ -19,6 +19,7 @@
 
   <ItemGroup>
     <PackageReference Include="AspNet.Security.OAuth.Discord" />
+    <PackageReference Include="Discord.Net" Version="2.4.0" />
```

Comment:
Remove the version specification. Also, probably remove this entirely. It hasn't been needed before, why would it be needed now?